tests/installation/user_settings.pm

```diff
@@ -35,16 +35,23 @@ sub run {
         return;
     }
 
-    # retry if not typed correctly
-    my $max_tries = 4;
-    my $retry     = 0;
-    do {
-        $self->enter_userinfo(max_interval => ($retry) ? utils::VERY_SLOW_TYPING_SPEED : undef);
+    if (check_var('SLE_PRODUCT', 'hpc')) {
```

Comment:
Actually I'm thinking how to add my case into this part, because it's a quite normal case (e.g. https://openqa.nue.suse.com/tests/4001162#settings) without any special parameter for filesystem. How about here just add a new parameter like `check_var('NO_TYPE_CHECK', 1)` here instand of you check hpc there? Then we add NO_TYPE_CHECK=1 in our installation job.

Reply:
@SeroSun, I also had the same idea: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9808#discussion_r393609289

Let's discuss a proper broader solution in #9808 
The discussion can take long and with this PR at least HPC is able to continue with their tests.